featurebyte/api/base_feature_target_namespace.py

```diff
@@ -0,0 +1,46 @@
+"""
+Base feature target namespace
+"""
+from typing import List
+
+from pydantic import Field
+
+from featurebyte.api.api_object import ApiObject
+from featurebyte.exception import RecordRetrievalException
+from featurebyte.models.base import PydanticObjectId
+from featurebyte.models.feature_namespace import BaseFeatureNamespaceModel, DefaultVersionMode
+
+
+class BaseFeatureTargetNamespace(ApiObject):
```

Comment:
Should we make it a mixin rather than a base class (similar to `feature_or_target_mixin.py`)?

Reply:
sure i can rename it, but just to clarify though, a mixin and a base class function pretty much the same way yea?

Reply:
From the code wise, both rely on python inheritance. There are some minor differences (https://stackoverflow.com/questions/860245/mixin-vs-inheritance):
Base class => Usually object is IS inheritance check
Mixin => mixin is rarely useful as a standalone object (it is like introducing special properties to the class)

Reply:
In this case, it is more like documentation purpose. If a class has name `Mixin`, we don't expect it to be instantiated and it should be inherited by some other classes.